### DISCUSSION TITLE
英雄无敌3死亡阴影下载连接失效了，求补。

### DISCUSSION THREAD

**Discussion by 342655596** [Gamuxorg/bbs#301] [游戏或网站运行反馈]
如题，死亡阴影连接失效了。

**Comment by 342655596** [comment]
> 如题，死亡阴影连接失效了。

**Comment by shouhuanxiaoji** [comment]
您好，我在贴吧也看到有人留言了，我还在回想之前是怎么打包的。。

**Comment by 342655596** [comment]
> 您好，我在贴吧也看到有人留言了，我还在回想之前是怎么打包的。。

那个已经不需要了，谢谢。wine乱码的问题，我自己解决了。贴吧那个是我留言的。

**Comment by 342655596** [comment]
> 您好，我在贴吧也看到有人留言了，我还在回想之前是怎么打包的。。

用wine就能玩了，乱码的问题我自己解决了。

**Comment by shouhuanxiaoji** [comment]
麻烦问一下，这个游戏现在玩的人多么，多的话我可以打包一份带wine的成appiamge上去

**Comment by 342655596** [comment]
> 麻烦问一下，这个游戏现在玩的人多么，多的话我可以打包一份带wine的成appiamge上去

我不清楚，还有一些老玩家就是了。B站也有一些这个游戏的新视频。

**Comment by shouhuanxiaoji** [comment]
em..那可以弄一下 我周日尝试一下
